webpack.config.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+    new CopyWebpackPlugin({
+      patterns: [
+        {
+          from: path.resolve(__dirname, 'src/images'),
```

Comment:
- [ ] [OPTIONAL] It seems that there is an issue with locating the 'images' folder in the 'src' directory when running the application. However, according to the webpack configuration file (`webpack.config.js`), the 'images' folder should be present in the 'src' directory.

To resolve this issue, please double-check the following:

1. Ensure that the 'images' folder exists in the 'src' directory.
2. Confirm that the 'images' folder contains the necessary image files.
3. Verify that the 'images' folder and its contents are correctly spelled and capitalized.

If the issue persists, please provide additional information or the relevant code snippet, so we can assist you further. Good luck!


<img width="958" alt="image" src="https://github.com/yashodhicy/LeaderBoard/assets/52934128/384b12bd-0df7-4d9c-a223-17e8e94613fc">